constellation_base/models.py

```diff
@@ -39,3 +42,20 @@ def settings_dict(self):
             'organization': self.organization,
             'titlebar_links': self.titlebar_links,
         }
+
+
+class LocalGroupACL(models.Model):
+    """ACL for Local Groups that don't exist in other authentication sources"""
+    group = models.ForeignKey(Group)
```

Comment:
...  Because this is the way that we track groups across the rest of the networks??  IMHO this is the standard way to do it.  If you do it the other way you wind up with a stupidly large number of rows in the database that need to be traversed to perform a membership update as well as added management complexity.